### ISSUE TITLE
CLI function to remove an alias

### ISSUE THREAD

**Issue by devoid** [ModelSEED/ModelSEED#78]
Not sure what this should be called. `ms remove biochemistry/sdevoid/main` ? `ms rm`? It should remove the alias from the set of aliases returned by `ms list` and all other commands. It should not actually delete the object. Object deletion is handled during Database deletion...at least for now.

**Comment by cshenry** [comment]
Agreed. We really need this. My db has gotten pretty cluttered now.
Chris

**Comment by pfrybar** [comment]
It will be easy to remove the alias and leave the object hanging... but it will take some additional logic to see if we can actually delete the object and its ancestors. For now we could just do the former and call it good.

**Comment by cshenry** [comment]
The former seems fine. I suggest we do the final deletion in a DB-wide clean up effort run separately and periodically.
